### PR TITLE
Added support for registering keybindings to open the quick add dropdown component

### PR DESCRIPTION
**Pull request by johntrickett86**
@awcodes Managed to figure it out!

Please review and let me know your thoughts.

### CODE REVIEW

**Comment by johntrickett86** [comment]
@awcodes It would have been nice to add a tooltip showing keyboard shortcut but I couldn't see a clear way to achieve that using the button component.

File: resources/views/components/create-menu.blade.php [hunk 1]
```diff
@@ -1,41 +1,51 @@
...
-                    :wire:click="$resource['action']"
-                    :href="$resource['url']"
-                    :tag="$resource['url'] ? 'a' : 'button'"
+@php
+    $keyBindings = collect($keyBindings ?? [])
+        ->map(fn (string $keyBinding): string => str_replace('+', '-', $keyBinding))
+        ->implode('.');
+@endphp
+
+<div class="quick-create-component" x-data="{ dropdownOpen: false }">
```
Comment (awcodes) [inline resources/views/components/create-menu.blade.php]:
```
@if($keyBindings)
    x-data="{
        toggleDropdown() {
            this.$refs.triggerButton.click()
            this.$refs.triggerButton.focus()
        }
    }"
@endif
```

File: resources/views/components/create-menu.blade.php [hunk 2]
```diff
@@ -1,41 +1,51 @@
...
+    @if ($resources && $this->shouldBeHidden() === false)
+        <x-filament::dropdown placement="bottom-end" :teleport="true">
+            <x-slot name="trigger">
+                <button
+                        x-ref="triggerButton"
```
Comment (awcodes) [inline resources/views/components/create-menu.blade.php:21]:
```
@if($keyBindings)
    x-ref="triggerButton"
    x-mousetrap.global.{{ $keyBindings }}="toggleDropdown"
@endif
```

**Review by awcodes — CHANGES_REQUESTED** [review]

**Comment by johntrickett86** [comment]
@awcodes Thanks for the feedback. That's been updated.

**Review by awcodes — APPROVED** [review]

**Pull request merged**